index.html

```diff
@@ -1,57 +1,287 @@
+  <div class="date">
+  <ul>
+  <li>Canopy</li>
+  <li><img src="images/dot.png" alt=""></li>
+  <li>Back End Dev</li>
+
+  <li><img src="images/dot.png" alt=""></li>
+  <li>2015</li>
+
+  </ul>
+
+  </div>
+  <div class="summary">
+  <h2>A daily selection of privately personalized reads; no accounts or sign-ups required.</h2>
+  </div>
+
+  <div class="languages">
+
+
+  <ul>
+  <li>html</li>
+  <li>Css</li>
+  <li>javascript</li>
+
+  </ul>
+
+
+
+  </div>
+
+  <div class="see-project">
+  <button>See Project</button>
+  </div>
+
+  </div>
+
+
+  </section>
+
+  <br>
+  <br>
+  <br>
+
+  <section>
+
+  <div class="snapshot-container">
+
+  <div class="snapshot"><img src="images/Snapshoot Portfolio4.png" alt=""></div>
+  <div class="pheading">
+  <h1>Multi-Post Stories</h1>
+  </div>
+  <div class="date">
+  <ul>
+  <li>Canopy</li>
+  <li><img src="images/dot.png" alt=""></li>
+  <li>Back End Dev</li>
+
+  <li><img src="images/dot.png" alt=""></li>
+  <li>2015</li>
+
+  </ul>
+
+  </div>
+  <div class="summary">
+  <h2>A daily selection of privately personalized reads; no accounts or sign-ups required.</h2>
+  </div>
+
+  <div class="languages">
+
+
+  <ul>
+  <li>html</li>
+  <li>Css</li>
+  <li>javascript</li>
+
+  </ul>
+
+
+
+  </div>
+
+  <div class="see-project">
+  <button>See Project</button>
+  </div>
+
+  </div>
+
+
+  </section>
+
+  <br>
+  <br>
+  <br>
+
+  <section>
+
+  <div class="snapshot-container">
+
+  <div class="before-about-space">
+
+  </div>
+
+  <div class="pheading">
+  <h1>About
+  Myself</h1>
+  </div>
+
+  <div class="summary-about">
+  <h2>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+  work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+  </h2>
+  </div>
+
+  <div class="lets-connect">
+  <h5>LET'S CONNECT</h5>
+  </div>
+
+  <div class="social-icons">
+  <ul>
+  <li><a href="https://twitter.com"><img src="images/twitter.png" alt="twitter"></a></li>
+  <li><a href="https://Linkedin.com"><img src="images/Linkedin.png" alt="Linkedin"></a></li>
+  <li><a href="https://medium.com"><img src="images/medium.png" alt="medium"></a></li>
+  <li><a href="https://github.com"><img src="images/github.png" alt="github"></a></li>
+  <li><a href="https://angellist.com"><img src="images/victory.png" alt="angellist"></a></li>
+  </ul>
+  </div>
+
+
+  </div>
+
+  <div class="about-see-project">
+  <button>Get my resume</button>
+  </div>
+
+  </div>
+
+
+  </section>
+  <br>
+  <br>
+  <section>
+  <div class="snapshot-container">
+
+  <div class="drowdown">
+  <h1>Languages</h1>
+  <i class="fa fa-angle-right" style="font-size:36px"></i>
+  </div>
+
+  <div class="framework">
+  <h1>Framework</h1>
+  <i class="fa fa-angle-right" style="font-size:36px"></i>
+  </div>
+  <div class="skills">
+  <h1>Skills</h1>
+  <i class="fa fa-angle-right" style="font-size:36px"></i>
+  </div>
+
+  </div>
+
+  </section>
```

Comment:
- [ ] please note that all of the about me section should be wrapped in **ONE** `<section>` and should be aligned with flexbox